libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Executables.kt

```diff
@@ -2,19 +2,15 @@
 
 package com.apollographql.apollo3.api
 
+import com.apollographql.apollo3.annotations.ApolloInternal
 import com.apollographql.apollo3.api.json.BufferedSinkJsonWriter
 import com.apollographql.apollo3.api.json.MapJsonWriter
 import okio.Buffer
 import kotlin.jvm.JvmName
 
 @Suppress("UNCHECKED_CAST")
 fun <D : Executable.Data> Executable<D>.variables(customScalarAdapters: CustomScalarAdapters): Executable.Variables {
-  val valueMap = MapJsonWriter().apply {
-    beginObject()
-    serializeVariables(this, customScalarAdapters.serializeVariablesWithDefaultBooleanValues())
-    endObject()
-  }.root() as Map<String, Any?>
-  return Executable.Variables(valueMap)
+  return variables(customScalarAdapters, true)
```

Comment:
SHould be a `false` here to not send the `false` over the network? 

```suggestion
  return variables(customScalarAdapters, false)
```

Reply:
For the network, `serializeVariables` is called directly (with a `CustomScalarAdapters` that has it to false), right?

Reply:
If you're using `apollo-runtime` yes but if you're not, I don't know what people are doing, I'd keep the behaviour without booleans for public methods. It'd be surprising to have default booleans in there

Reply:
Ah yes that makes sense! So one last (hopefully!) tweak.